tests/wallet/nft_wallet/test_nft_1_offers.py

```diff
@@ -55,14 +55,16 @@ async def get_nft_count(wallet: NFTWallet) -> int:
     [True, False],
 )
 @pytest.mark.parametrize(
-    "forwards_compat",
-    [True, False],
+    "forwards_compat_and_zero",
+    [(True, False), (False, False), (False, True)],
```

Comment:
you can make these separate parameters to the test function directly, by:
```
    "forwards_compat,zero",
    [(True, False), (False, False), (False, True)],
```

and:

```
async def test_nft_offer_sell_nft(
    self_hostname: str, two_wallet_nodes: Any, trusted: Any, forwards_compat: bool, zero: bool
) -> None:
```